exposed-tests/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
     implementation("org.jetbrains.kotlinx", "kotlinx-coroutines-core", "1.3.0-M1")
     implementation(project(":exposed-core"))
     implementation(project(":exposed-jdbc"))
+    implementation(project(":exposed-jodatime"))
```

Comment:
Is that dependency really needed?

Reply:
@Tapac for instance no
but when I will add tests on objects that contains datetime columns i will need it.
anyway we can delete. I will add it in my next PR.

Reply:
Ah ok I understand now 😄  ! there is a test forder in exposed-java-time and exposed-jodatime.
I will add my tests there. So no need for this dependency 👍 
I aldready removed it.